store/tikv/cleanup.go

```diff
@@ -36,37 +37,41 @@ func (actionCleanup) tiKVTxnRegionsNumHistogram() prometheus.Observer {
 	return tiKVTxnRegionsNumHistogramCleanup
 }
 
-func (actionCleanup) handleSingleBatch(c *twoPhaseCommitter, bo *Backoffer, batch batchMutations) error {
+func (actionCleanup) collectMutation(acc *CommitterMutations, m *mutation) {
```

Comment:
What does `acc` mean?

Reply:
Accumulator, In FP style it is often abbreviated to `acc`.